platform/src/pages/index.js

```diff
@@ -4,7 +4,12 @@ function HomePage() {
       <h1>Airqo platform (beta)</h1>
       <div>
         Find the original airqo platform
-        <a href={process.env.NEXT_PUBLIC_ORIGINAL_PLATFORM}>here</a>
+        <a
+          className='text-3xl p-2 underline font-bold'
+          href={process.env.NEXT_PUBLIC_ORIGINAL_PLATFORM}
```

Comment:
Can we have a file for exporting environment constants? 
In order to have some abstraction and avoid using **process.env** in the UI components (makes it easy for one to inspect the UI and know how we name our env variables)